go.mod

```diff
@@ -18,6 +20,7 @@ require (
 	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
 	google.golang.org/grpc v1.40.0
 	google.golang.org/protobuf v1.27.1
+	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
```

Comment:
This shouldn't be needed anymore, missing `go mod tidy` ?

Reply:
done